economics.toml

```diff
@@ -20,26 +20,35 @@
 
 [RewardsSettings]
     [[RewardsSettings.RewardsConfigByEpoch]]
-    EpochEnable = 0
-    LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
-    DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
-    ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
-    ProtocolSustainabilityAddress = "erd1j25xk97yf820rgdp3mj5scavhjkn6tjyn0t63pmv5qyjj7wxlcfqqe2rw5"
-    TopUpGradientPoint = "3000000000000000000000000" # 3MIL eGLD (eligible topUp)
-    TopUpFactor = 0.25 # fraction of value 0.25 - 25%
+        EpochEnable = 0
+        LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
+        DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
+        ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
+        ProtocolSustainabilityAddress = "erd1j25xk97yf820rgdp3mj5scavhjkn6tjyn0t63pmv5qyjj7wxlcfqqe2rw5"
+        TopUpGradientPoint = "3000000000000000000000000" # 3MIL eGLD (eligible topUp)
+        TopUpFactor = 0.25 # fraction of value 0.25 - 25%
 
     [[RewardsSettings.RewardsConfigByEpoch]]
-    EpochEnable = 1
-    LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
-    DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
-    ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
-    ProtocolSustainabilityAddress = "erd1j25xk97yf820rgdp3mj5scavhjkn6tjyn0t63pmv5qyjj7wxlcfqqe2rw5"
-    TopUpGradientPoint = "2000000000000000000000000" # 2MIL eGLD (eligible topUp)
-    TopUpFactor = 0.5 # fraction of value 0.5 - 50%
+        EpochEnable = 1
+        LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
+        DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
+        ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
+        ProtocolSustainabilityAddress = "erd1j25xk97yf820rgdp3mj5scavhjkn6tjyn0t63pmv5qyjj7wxlcfqqe2rw5"
+        TopUpGradientPoint = "2000000000000000000000000" # 2MIL eGLD (eligible topUp)
+        TopUpFactor = 0.5 # fraction of value 0.5 - 50%
+
+    [[RewardsSettings.RewardsConfigByEpoch]]
```

Comment:
I don't think it's necessary to have this many `RewardsSettings.RewardsConfigByEpoch` 😄

Reply:
+1